fixtures/american-dj/mini-dekker.json

```diff
@@ -0,0 +1,101 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Mini Dekker",
+  "shortName": "ADJ Mini Dekker",
+  "categories": ["Effect"],
+  "meta": {
+    "authors": ["David"],
+    "createDate": "2023-12-06",
+    "lastModifyDate": "2023-12-06"
+  },
+  "links": {
+    "manual": [
+      "http://adjmedia.s3-website-eu-west-1.amazonaws.com/manuals/Mini%20Dekker_FR%2005152019.pdf"
+    ],
+    "productPage": [
+      "https://www.thomann.de/fr/adj_mini_dekker.htm"
```

Comment:
This is merchant's page.  The manufacturer page is: https://www.adj.com/mini-dekker